### PR TITLE
Remove offending test

### PR DESCRIPTION
**Pull request by mftcodes**
## Change Details

* Removing a test in a golang file that exposed the unencrypted bfdHashPepper for PROD_SBX

## Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [ ] PHI/PII is affected by this change

### CODE REVIEW

**Review by famegashie — DISMISSED** [review]
Reviewed.

**Review by lgarceau768 — CHANGES_REQUESTED** [review]
I think the version upgrades should be moved to a new PR but my mind can be changed!

File: dpc-web/Gemfile [hunk 1]
```diff
@@ -7,17 +7,17 @@ ruby '2.7.2'
 
 # Anchored versions, do not bump without testing
```
[the remark this answers is not in the document]
Comment (mftcodes) [reply dpc-web/Gemfile:8]: I can separate a PR to upgrade all of these, and a PR to remove the test. All the upgraded packages are necessary to even get a build to finish. Is this the separate you mean?

**Review by lgarceau768 — APPROVED** [review]
Reviewed and looks good!

**Pull request merged**